conventions.md

```diff
@@ -0,0 +1,15 @@
+# Conventions
+
+A basic overview of (semi-)arbitrary conventions we follown in this repository.
+
+## Closed channel error handling
+
+We handle closed channel errors in two ways.
+
+### expect
+
+If we have a guarantee that `rx` is still open during send (for example we own the `rx`), we call `.expect(...)` on the `Result` of send.
+
+### map_err
```

Comment:
You could write here a bit more about what the guarantees/expectations are – i.e. whether this is for channels that have the other end outside of AlephBFT or some others too and why we don't panic on these.

Shouldn't we actually? for many of them it seems that if they are closed then continuing to run the consensus might be wrong? Or perhaps the idea is that the rest of the node fails, but at least it supports the consensus? But the latter wouldn't work anyway, because of data availability, would it?

This definitely needs a tad more explanation, but also some discussion as to what our actual policy should be.

Reply:
You are right, this map_err in some cases is obsolete, so I changed it to always panic on other side of channel being closed